packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -101,6 +105,10 @@ export default decorate([
           return
         }
         const resourceParams = await form({
+          defaultValue: {
+            mapPoolsSrs: {},
```

Comment:
`mapPoolSrs` is not used in this case.

Reply:
But it's needed in order for the component bellow works fine for the two cases.

Reply:
Otherwise, I create a component for each case.

Reply:
The components bellow will work fine because they don't depend on this entry.

Reply:
According to my algorithm, It won't work fine because I always need an initial value.

Reply:
This [condition](https://github.com/vatesfr/xen-orchestra/pull/4571/files#diff-19f1a84d6035c2fa112b404861afeab9R62) handle the other cases.